user-guide/howto/files/pxe/netboot_install_script.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash
+#
+# The logic is simple:
+#  1. Download and extract USB image into /tmp/usbkey
+#  2. Run prompt-config.sh, which will create /usbkey on the zones pool
+#  3. Copy contents of /tmp/usbkey into /usbkey
+#
+# There is one little problem:
+#   The original prompt-config.sh does a reboot before it finishes =>
+#   => we need to change that before running the script.
```

Comment:
Fixed in https://github.com/erigones/esdc-factory/commit/f5a06832c98b49a9c51d6b7077777401d6cc5589